test/native/test_build_config.cpp

```diff
@@ -4,14 +4,14 @@
  * Unit tests of the build_config.hpp header.
  */
 
-#include "uavcan/uavcan.hpp"
+#include "libuavcan/uavcan.hpp"
```

Comment:
Perhaps the main header should be renamed, too? `libuavcan/libuavcan.hpp`?

Reply:
I thought about that but I *think* this is valid as: "include uavcan from libuavcan." Put another way: if libuavcan had a different name, say "pavel" then this would be `#include "pavel/uavcan.hpp"` or "include uavcan from the pavel library". 

I dunno. What do you think?

Reply:
I'm thinking that the main header should be eponymous with the namespace of the library. But I don't have any strong preferences here either way.

Reply:
> But I don't have any strong preferences here either way.

This is not to say that I am not inclined to prefer `libuavcan/libuavcan.hpp`. I am going to mark this as approved nevertheless but unless you disagree it's best to rename it.